src/core/connectivity.jl

```diff
@@ -709,9 +800,11 @@ end
 """
     get_adjacency_list(connectivity::AbstractConnectivity)::Dict{Int,Vector{Int}}
 
-Given an object of type `AbstractConnectivity`, `get_adjacency_list` returns a Dict where `key => value` pairs
-are each qubit number => a Vector of the qubits that are adjacent (neighbors) to it on this particular connectivity.
-Positions in `connectivity.excluded_positions` are not given a key in the adjacency list.
+Given an object of type `AbstractConnectivity`, `get_adjacency_list` returns a Dict with
+`key => value` pairs. Each key corresponds to a qubit index while each value is a `Vector`
+of the qubit indices which are adjacent (neighbors) to the qubit in the `key`. The adjacency
```

Comment:
For each qubit key, a vector with all the qubits that are adjacent to it on that connectivity, i.e., a connection exist between them, is returned as a Vector.

Reply:
I decided to use:
```
Given an object of type `AbstractConnectivity`, `get_adjacency_list` returns a `Dict` where
each key is a qubit index. Every dictionary value is a `Vector` that lists all the qubits
which are adjacent on the connectivity to the qubit key. Positions in
`connectivity.excluded_positions` are not included as keys nor values.
```